src/features/resolve.js

```diff
@@ -1,10 +1,11 @@
 import { BASE_URL, baseUrl, resolveImportMap, resolveIfNotPlainOrUrl, IMPORT_MAP } from '../common.js';
+import { importMap } from './import-maps.js';
 import { systemJSPrototype } from '../system-core.js';
 import { errMsg } from '../err-msg.js';
 
 systemJSPrototype.resolve = function (id, parentUrl) {
   parentUrl = parentUrl || !process.env.SYSTEM_BROWSER && this[BASE_URL] || baseUrl;
-  return resolveImportMap(this[IMPORT_MAP], resolveIfNotPlainOrUrl(id, parentUrl) || id, parentUrl) || throwUnresolved(id, parentUrl);
+  return resolveImportMap((!process.env.SYSTEM_BROWSER && this[IMPORT_MAP] || importMap), resolveIfNotPlainOrUrl(id, parentUrl) || id, parentUrl) || throwUnresolved(id, parentUrl);
```

Comment:
Could you help me understand the `|| importMap`? Is this so that NodeJS resolves of full urls work without requiring the user to applyImportMap first? If so, could this be done by setting `this[IMPORT_MAP]` = emptyMap?

Reply:
This is the same pattern we use for `baseUrl` between the Node.js and browser versions.